conda/common/compat.py

```diff
@@ -131,7 +131,7 @@ def iteritems(d, **kw):
     viewvalues = methodcaller("values")
     viewitems = methodcaller("items")
 
-    from collections import Iterable
+    from collections.abc import Iterable
```

Comment:
This has to be wrapped in a try/except to keep py27 compat:

```suggestion
    try:
        from collections.abc import Iterable
    except ImportError:
        from collections import Iterable
```

Reply:
This code is inside of a `if PY3` block so there should be no need for py27 compatibility.  `collection.abc` is only needed for compatibility with Python 3.0-3.2.

Reply:
Thanks for expanding the context.  Seems good to go.